apis/replication.storage/v1alpha1/volumegroupreplicationcontent_types.go

```diff
@@ -68,16 +68,10 @@ type VolumeGroupReplicationContentSource struct {
 
 // VolumeGroupReplicationContentStatus defines the status of VolumeGroupReplicationContent
 type VolumeGroupReplicationContentStatus struct {
-	// PVVolumeList is the list of of PV for the group replication
+	// PersistentVolumeRefList is the list of of PV for the group replication
 	// The maximum number of allowed PV in the group is 100.
 	// +optional
-	PVVolumeList []PVVolume `json:"pvVolumeList,omitempty"`
-}
-
-// PVVolumes represent PV name to be groped and replicated.
-type PVVolume struct {
-	// PersistentVolumeRef is a reference to the persistent volume resource
-	PersistentVolumeRef corev1.LocalObjectReference `json:"persistentVolumeRef,omitempty"`
+	PersistentVolumeRefList []corev1.LocalObjectReference `json:"persistentVolumeRefList,omitempty"`
```

Comment:
ah, no, it is part of a different commit

Reply:
Please move this change to the other commit, that is the only thing pending that I can spot.

Reply:
sorry i missed it, pushed it to right commit